vespajlib/src/main/java/ai/vespa/json/Json.java

```diff
@@ -161,7 +165,8 @@ private void requireType(Type... types) {
     private void requirePresent() { if (isMissing()) throw createMissingMemberException(); }
 
     private InvalidJsonException createInvalidTypeException(Type... expected) {
-        var expectedTypesString = Arrays.stream(expected).map(this::toString).collect(Collectors.joining("' or '", "'", "'"));
+        var expectedTypesString = Arrays.stream(expected).map(this::toString)
+                .collect(java.util.stream.Collectors.joining("' or '", "'", "'"));
```

Comment:
Static import?

Reply:
Conflicting with the static subclass `Collectors` that I introduced